scripts/ci/Dockerfile.bundle-test-22-04

```diff
@@ -0,0 +1,20 @@
+# Copyright Kani Contributors
```

Comment:
nit: Rename file to include ubuntu-22-04

Reply:
Do you mean adding "ubuntu" to all 3 file names here?

Reply:
Yes, I mean adding Ubuntu name to this new file as well as the 18-04 and the 20-04. That seems like the naming convention you are following `Dockerfile.bundle-test-<os>`, right?